packages/web-react/src/components/Tooltip/TooltipTrigger.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ElementType } from 'react';
+import { useStyleProps } from '../../hooks';
+import { StyleProps } from '../../types';
+import { useTooltipContext } from './TooltipContext';
+
+interface TooltipTriggerProps extends StyleProps {
+  elementType?: ElementType | string;
+  children: string | React.ReactNode | ((props: { isOpen: boolean }) => React.ReactNode);
```

Comment:
```suggestion
  children: string | ReactNode | ((props: { isOpen: boolean }) => .ReactNode);
```

Please, import the `ReactNode` from the `React`.

Reply:
done